src/layout/Input/InputComponent.tsx

```diff
@@ -17,6 +18,8 @@ export function InputComponent({ node, isValid, formData, handleDataChange }: II
     formData?.simpleBinding ?? '',
     saveWhileTyping,
   );
+
+  const reactNumberFormatConfig = useMapToReactNumberConfig(value, formatting as IInputFormatting);
```

Comment:
Why do we need to cast the formatting to be the type IInputFormatting here? Is formatting from node.item something else? :)

Reply:
I get a Typescript warning if I remove it. Undefined is not assignable to type IInputFormatting - this parameter in useMapToReactNumberConfig has also the type IInputFormatting. I'm not that solid in Typescript yet, maybe you see a solution`?